core/src/core/usb.zig

```diff
@@ -226,12 +227,22 @@ pub fn Usb(comptime f: anytype) type {
                                     S.configured = true;
                                     CmdEndpoint.send_cmd_ack();
                                 },
-                                SetupRequest.GetDescriptor => {
+                                .GetDescriptor => {
                                     const descriptor_type = DescType.from_u8(@intCast(setup.value >> 8));
                                     if (descriptor_type) |dt| {
                                         try process_get_descriptor(setup, dt);
                                     }
                                 },
+                                .SetFeature => {
+                                    const feature = FeatureSelector.from_u8(@intCast(setup.value >> 8));
+                                    if (feature) |feat| {
+                                        switch (feat) {
+                                            .DeviceRemoteWakeup, .EndpointHalt => CmdEndpoint.send_cmd_ack(),
+                                            // TODO
```

Comment:
Let's file an issue here for any follow up and put the issue number in the comment

Reply:
Upon further investigation, "test mode" seems to be a way to test the integrity of the USB hardware itself and is only sent by testing devices, not typical hosts. I can't tell if this is something that could even be implemented in firmware on a RP2040 (for example).

I can still make an issue to investigate more, regardless.